packages/node_modules/nav-frontend-lenker-style/src/lenker-mixins.less

```diff
@@ -25,3 +28,20 @@
     color: white;
   }
 }
+
+.lenkemedikon-mixin {
+  svg {
+    width: 1rem;
```

Comment:
Hm, burde størrelsene for ikonet være definert med `em` i stedet for `rem` her? Tenker det kan være fint om dette skalerer med font-størrelsen 🤔

Hvordan vil f.eks. dette funke:

```jsx
<Undertittel>
    <Lenke href="#">
        <span>lenke med ikon</span>
        <SvgLinkIcon />
    </Lenke>
</Undertittel>
```